pages_deployment.go

```diff
@@ -181,6 +210,32 @@ func (api *API) GetPagesDeploymentStageLogs(ctx context.Context, params GetPages
 	return r.Result, nil
 }
 
+// GetPagesDeploymentStageLogs returns the logs for a Pages deployment stage.
+//
+// API reference: https://api.cloudflare.com/#pages-deployment-get-deployment-stage-logs
+func (api *API) GetPagesDeploymentLogs(ctx context.Context, params GetPagesDeploymentLogsParams) (PagesDeploymentLogs, error) {
+	v := url.Values{}
+	if params.Size > 0 {
+		v.Set("size", strconv.Itoa(params.Size))
+	}
```

Comment:
we don't need to manually do this if we use the `url` tags in the structs followed by [`buildURI`](https://github.com/cloudflare/cloudflare-go/blob/e5c6a2d8740459c045a4f020b5510a87b29fa548/utils.go#L10) passing in the values we need here. check out https://github.com/cloudflare/cloudflare-go/blob/master/tunnel_routes.go#L31-L41 for some example usage of this.